src/foam/nanos/servlet/ResourceImageServlet.java

```diff
@@ -49,7 +49,8 @@ protected void service(HttpServletRequest req, HttpServletResponse resp)
       String ext = EXTS.get(request.substring(request.lastIndexOf('.') +1));
       try ( BufferedInputStream bis = new BufferedInputStream(is) ) {
         resp.setContentType(!SafetyUtil.isEmpty(ext) ? ext : DEFAULT_EXT);
-        resp.setHeader("Content-Disposition", "filename=\"" + StringEscapeUtils.escapeHtml4(request) + "\"");
+        // resp.setHeader("Content-Disposition", "filename=\"" + StringEscapeUtils.escapeHtml4(request) + "\"");
```

Comment:
Why did you remove this, not that I remember what is was for.

Reply:
Oh, I forgot to mention that in my PR description. As far as I can tell this header is incorrect so I decided to comment it out while I was testing the OverlayActionListView icons. According to the MDN page the Content-Disposition header should begin with `inline` or `attachment` and filename should be just a name for the file if it's downloaded (right now it's a path)

Reply:
Yes, the header format is incorrect, but also, I don't think it makes sense. Look at the commit for when this was introduced to see why the developer added it. If no good reason is presented, then remove in a separate PR which explains why.

Reply:
It was there from the inception of the class, so simply originally wrong .